README.md

```diff
@@ -1,85 +1,34 @@
-# Technical Documentation
+# Tech docs manual
 
-## Getting started
+This project contains the documentation for the GOV.UK Tech Docs project.
 
-To preview or build the website, we need to use the terminal.
+👉 [Go to alphagov.github.io/tech-docs-manual](https://alphagov.github.io/tech-docs-manual)
 
-Install Ruby with Rubygems, perferably with a [Ruby version manager][rvm],
-and the [Bundler gem][bundler].
+It uses the [tech-docs-gem](https://github.com/alphagov/tech-docs-gem) for layout. To avoid duplication between that gem and this repo, it pulls in certain documentation from the gem into this project. 
```

Comment:
I think this paragraph is a bit confusing if you're looking to understand how the tech docs gem/template work, rather than the tech docs manual, which itself uses the tech docs.

Can we move it under a dependencies subheading?